config/locales/en.yml

```
@@ -410,6 +411,15 @@ en:
         infinite_scroll: Infinite scroll
         load_more_button: "'Load more' button"
 
+  linked_users:
+    description: Link your account together to allow log in to the same account using different logins. You can log in using different platforms with as many account as you wish, and all of them will log you in to one main account.
```

I think this could be worded a little differently:
```suggestion
    description: Link your accounts together to allow login to the same account using different authentication platforms. You can link as many accounts from any platform as you want, and all of them will log you into one central account.
```